products/jbrowse-react-linear-genome-view/src/corePlugins.ts

```diff
@@ -8,6 +8,7 @@ import Sequence from '@jbrowse/plugin-sequence'
 import SVG from '@jbrowse/plugin-svg'
 import Variants from '@jbrowse/plugin-variants'
 import Wiggle from '@jbrowse/plugin-wiggle'
+import LegacyJBrowse from '@jbrowse/plugin-legacy-jbrowse'
```

Comment:
This plugin will need to be added to the package.json. Also, we don't publish the legacy JBrowse plugin on NPM yet, so we'll need to do that as well (basically just changing the package.json of that plugin to make it public so the next release picks it up).

Reply:
is this resolved now?

Reply:
Added that here https://github.com/GMOD/jbrowse-components/pull/2045/commits/bc5bda3d7cd162b3dcade592482c30fc6ffec77f